TrackletAlgorithm/MatchProcessor_parameters.h

```diff
@@ -1,115 +1,170 @@
 #ifndef TrackletAlgorithm_MatchProcessor_parameters_h
```

Comment:
@carriganm95 I'd like to avoid having duplicate code if possible. Is there any way to adapt the workflow for the emulation so that you use the version of this file produced by generate_MP.py?

Reply:
yes we can do that